### DISCUSSION TITLE
quick start: python package error after rez-bind --quickstart

### DISCUSSION THREAD

**Discussion by frueter** [AcademySoftwareFoundation/rez#1448] [Q&A]
Hi,
I am just trying to go through the very first steps of rez and was trying to build the hello_world example package.
However it failed with this error:
```
Running build command: python C:\Users\frank\Downloads\rez\example_packages\hello_world/build.py install
18:11:19 ERROR    PackageCommandError: Error in post_commands in package 'c:\\users\\frank\\packages\\python\\3.10.9\\package.py[0]':
[WinError 3] The system cannot find the path specified: 'c:\\users\\frank\\packages\\python\\3.10.9\\platform-windows\\arch-AMD64\\os-windows-10.0.22000.SP0\\python'
```
And sure enough, there is no "python" folder in c:\\users\\frank\\packages\\python\\3.10.9\\platform-windows\\arch-AMD64\\os-windows-10.0.22000.SP0, only a "bin" folder.

I believe the python package was created by the `rez-bind --quickstart` command that is listed in teh quick start docs.

Since this is the very first time I am trying to use rez I have no idea where to turn from here, could somebody help please?
This is on Windows 11

Cheers,
frank

**Comment by instinct-vfx** [comment]
Hey Frank,

rez-bind is to be deprectated and specifically the python bind does not work well on windows if at all. I would suggest using it only to create the implicit packages (by running `rez-bind os` that should also pull in the other two as dependencies and creating a python package with actual payload. You can use a script i created for a tutorial a while back that you can find here: https://github.com/techartorg/rez_utils/tree/main/rezify_python.

It downloads the python version you request using nuget and creates a functional self-contained python package.

**Comment by instinct-vfx** [comment]
@AcademySoftwareFoundation/rez-maintainers  Note to self: We really need to get up notes/warnings for rez-bind on windows.

**Comment by frueter** [comment]
Thanks Thorsten,

You did mention these things to me before I got started,  and I did try to
follow your advise, but the correlations were not clear to me at this
stage. I saw no mention of rez-bind being deprecated anwhere in the docs,
only in your tutorial, and after only binding os,  python was not found at
all, so I tried using --quicksand despite you advising against it, as I
wasn't sure how rezify_python fit into the picture yet (I am a complete
beginner and still need to figure out the basics despite having read a fair
bit by now).
If does feel like the quick start guide needs a bit of work to assist
people actually getting started on windows (quickly or slowly ;)).

Will try again next week.

Cheers,
Frank

**Reply by instinct-vfx** [reply]
Sadly there is no mention indeed and you are not the first to hit that entry barrier so we do need to address it.
